File: mg/php/mg/inst.php
<?php
require 'vendor/autoload.php';

$client=new MongoDB\Client("mongodb://localhost:27017");
$db=$client->phpdb2;

$col=$db->usr;

$res=$col->insertOne(['name'=>"ancel","age"=>20]);

echo "result : ".$res->getInsertedId();

$res2=$col->insertMany([['name'=>'sujal','age'=>21],['name'=>'shubham','age'=>19]]);

foreach($res2->getInsertedIds() as $id){
    echo " result : ".$id."<br>";
}
?>

File: mg/py/inst.py
from pymongo import MongoClient

client= MongoClient("mongodb://localhost:27017")

db=client["pydb1"]
col=db["usr"]

records=[
    {"name":"ancee","age":20},
    {"name":"sujal","age":21},
    {"name":"chaitu","age":21}
    ]

i=col.insert_many(records);
print("inserted is : ",i.inserted_ids);

for doc in col.find():
    print(doc)